gstreamer/classify.py

```diff
@@ -51,27 +72,29 @@ def main():
     args = parser.parse_args()
 
     print("Loading %s with %s labels."%(args.model, args.labels))
-    engine = ClassificationEngine(args.model)
+    interpreter = common.make_interpreter(args.model)
+    interpreter.allocate_tensors()
     labels = load_labels(args.labels)
 
-    input_shape = engine.get_input_tensor_shape()
-    inference_size = (input_shape[1], input_shape[2])
-
+    w, h, _  = common.input_size(interpreter)
+    inference_size = (w, h)
     # Average fps over last 30 frames.
-    fps_counter  = common.avg_fps_counter(30)
+    fps_counter = common.avg_fps_counter(30)
 
     def user_callback(input_tensor, src_size, inference_box):
       nonlocal fps_counter
       start_time = time.monotonic()
-      results = engine.classify_with_input_tensor(input_tensor,
-          threshold=args.threshold, top_k=args.top_k)
+      common.set_interpreter(interpreter, input_tensor)
+      # For larger input tensor sizes, use the edgetpu.classification.engine for better performance
```

Comment:
larger batches, larger image sizes, or both?

Reply:
Jonas suggested a comment saying that "for larger input tensors one API is currently a bit faster than the other" (that the old API would perform better than TFLite). To my knowledge this would be at least larger image sizes, what do you mean by batches?

Thanks!

Reply:
https://github.com/google-coral/examples-camera/blob/d1ddb17d0a2c259089e1ce4d2768954492b1f67e/pygame/classify_capture.py#L43 .  I assumed the _ here was for the number of batches, i.e., classifying more than one image at a time.

Coral i think is better for smaller batch sizes?  But i'm not sure, which is why i was asking :)

Reply:
I'll change the comment to say 'For larger input image sizes..." for clarity